docs/api/selector.md

```diff
@@ -12,6 +12,16 @@ follows:
 - class syntax (`.foo`, `.foo-bar`, etc.)
 - tag syntax (`input`, `div`, `span`, etc.)
 - id syntax (`#foo`, `#foo-bar`, etc.)
+- attr syntax(`[foo="bar"]`, `[foo]`, etc.);
+
+A note on attribute selectors: For compatibilty, you can use either the
+react prop name, or the native html selector. For example, the following
```

Comment:
I just pushed up the changes that @ljharb requested. Hopefully this can help the conversation move along.

looking for feedback from @lelandrichardson and @ljharb